briefcase-forms.rst

```diff
@@ -0,0 +1,160 @@
+**********************************
+Briefcase Usage
+**********************************
+
+:doc:`Install Briefcase <briefcase-install>` before going through this section.
+
+Pulling Forms from ODK Aggregate
+---------------------------------
+
+- Open *ODK Briefcase*.
+- In the window which opens up select :guilabel:`pull` tab.
+- Select Aggregate 0.9.x or Aggregate 1.0 in the :guilabel:`pull data from` drop-down.
+- Enter the url of your ODK Aggregate server in :guilabel:`url` option. For instance `https://proj1.appspot.com` and click on connect.
+- You can either leave the username blank if the anonymousUser is granted Data Viewer permissions, or specify an ODK Aggregate user (Account Type 'ODK') that has been granted Data Viewer permissions.
+- A list of forms will be displayed. Select the ones you want to pull and click on :guilabel:`pull` button at the bottom of window.The forms will be pulled to ODK Briefcase Storage on your local system. 
+
+.. warning::
+
+ ODK Briefcase will abort a pull attempt if the local copy of the form definition file differs in any way from the copy being pulled from the server or ODK Collect device.
+
+.. tip::
+
+ In the above case, if the changes only affect the question text and do not alter the structure of the collected data (or change the formId or version), you can create a new ODK Briefcase Storage area, pull data into that, then manually copy the instances from there into your original storage area.
```

Comment:
I think it's a little unclear what "in the above case" refers to. How about saying something like "If the form definition has changed but the changes only..."

Reply:
@ankita240796 Can you make this change?